src/config-schema.ts

```diff
@@ -67,12 +68,22 @@ const configSchema /*: JSONSchemaType<ConfigType> */ = {
   additionalProperties: false,
 };
 
+// We can't define the type of this variable,
+// see: https://github.com/ajv-validator/ajv/issues/1521
+const configFileSchema /*: JSONSchemaType<ConfigFileType> */ = {
+  type: 'object',
+  properties: {
+    additionalProperties: { $ref: 'muninn.configSchema.json' },
```

Comment:
`additionalProperties` property needs to be placed alongside the `properties`, not inside it.